src/util/util.cpp

```diff
@@ -694,6 +694,28 @@ bool pointInsidePolygon(vector<vec2>& poly, vec2 p) {
 	return inside;
 }
 
+void tga_write(const char* filename, uint32_t width, uint32_t height, uint8_t* data, uint8_t dataChannels, uint8_t fileChannels)

```

Comment:
Untested (for export obj with textures, later)

Reply:
Why TGA instead of BMP? All the asset-creation tools use BMP (Wally, sprite creation, model compiler).